Algorithm.CSharp/OrderTicketDemoAlgorithm.cs

```diff
@@ -473,6 +474,27 @@ private bool TimeIs(int day, int hour, int minute)
             return Time.Day == day && Time.Hour == hour && Time.Minute == minute;
         }
 
+        public override void OnEndOfAlgorithm()
+        {
+            var filledOrders = Transactions.GetOrders(x => x.Status == OrderStatus.Filled);
+            var openOrders = Transactions.GetOpenOrders(x => true);
```

Comment:
Cool! 
- `x => true` maybe can check the symbol is `X symbol`

- `Log()` suggest avoid logging if not required, and only using `Debug` when desired. For regression tests there is no real value for logging and causes issues really (there is a max output size, etc)